### PR TITLE
test(ci): github

### PR DESCRIPTION
**Pull request by bf4**
#### Additional helpful information

### CODE REVIEW

**Comment by Physium** [comment]
@bf4 would love to take a look at this and revive this. Is there a better way for me to update code directly to this branch as oppose to creating a PR to merge into this branch just to test CI?

**Comment by bf4** [comment]
@Physium I've invited you as a collaborator.  Looking forward to having your assistance:)

**Comment by Physium** [comment]
@bf4 Tweaked a couple of things to get the pipeline going. I removed 4.1 and 4.0 rails version from the matrix for now. Personally feel that its not worth the effort having this much backwards compatibility.

**Comment by Physium** [comment]
let me know whenever if its good to go or pending a review!

**Comment by bf4** [comment]
@Physium if it works for you,  it works for me.  :)

File: Gemfile [hunk 1]
```diff
@@ -36,6 +36,9 @@ if RUBY_VERSION < '2'
 end
 
 if ENV['CI']
+  # loofah has a breaking change 2.21 onwards which does not work with < 2.5 ruby
+  gem 'loofah', '< 2.21' if RUBY_VERSION < '2.5'
```
Comment (bf4) [inline Gemfile:40]: 👍

**Pull request merged**